spec/services/force/soql/preference_service_spec.rb

```diff
@@ -11,33 +11,60 @@
   describe 'buildAppPreferencesFilters' do
     it 'should create the correct accessibility string for Mobility' do
       VCR.use_cassette('services/soql/preference_service') do
-        filters = subject.buildAppPreferencesFilters({ listing_id: LEASE_UP_LISTING_ID, accessibility: 'Mobility impairments' })
+        filters = subject.buildAppPreferencesFilters({ listing_id: LEASE_UP_LISTING_ID, accessibility: ['Mobility impairments'] })
 
         expect(filters).to include("INCLUDES ('Mobility impairments')")
       end
     end
 
     it 'should create the correct accessibility string Vision/Hearing' do
       VCR.use_cassette('services/soql/preference_service') do
-        filters = subject.buildAppPreferencesFilters({ listing_id: LEASE_UP_LISTING_ID, accessibility: 'Vision impairments, Hearing impairments' })
+        filters = subject.buildAppPreferencesFilters({ listing_id: LEASE_UP_LISTING_ID, accessibility: ['Vision impairments',
+                                                                                                        'Hearing impairments'] })
 
         expect(filters).to include("INCLUDES ('Vision impairments', 'Hearing impairments')")
       end
     end
 
     it 'creates the expected query string when householdmembers is set to 1' do
       VCR.use_cassette('services/soql/preference_service') do
-        filters = subject.buildAppPreferencesFilters({ total_household_size: '1' })
+        filters = subject.buildAppPreferencesFilters({ total_household_size: ['1'] })
 
-        expect(filters).to include("and Application__r.Total_Household_Size__c = 1")
+        expect(filters).to include('and (Application__r.Total_Household_Size__c = 1)')
       end
     end
 
     it 'creates the expected query string when householdmembers is set to 5+' do
       VCR.use_cassette('services/soql/preference_service') do
-        filters = subject.buildAppPreferencesFilters({ total_household_size: '5+' })
+        filters = subject.buildAppPreferencesFilters({ total_household_size: ['5+'] })
 
-        expect(filters).to include("and Application__r.Total_Household_Size__c >= 5")
+        expect(filters).to include('and (Application__r.Total_Household_Size__c >= 5)')
+      end
+    end
+
+    it 'creates the expected query string when householdmembers has two values' do
+      VCR.use_cassette('services/soql/preference_service') do
+        filters = subject.buildAppPreferencesFilters({ total_household_size: ['1', '5+'] })
+
+        expect(filters).to include('and (Application__r.Total_Household_Size__c = 1 or Application__r.Total_Household_Size__c >= 5)')
+      end
+    end
+
+    it 'creates the expected query string when status has two values' do
```

Comment:
thanks for adding these. Can you add a test for the case that broke in the old version?

![100811373-5d45fc00-33ef-11eb-93ae-4fa989878d7a](https://user-images.githubusercontent.com/64036574/100934998-62b14e00-34a4-11eb-80cf-acadd7483e72.png)